spec/clearance/session_spec.rb

```diff
@@ -32,12 +32,12 @@
 
   context "with a custom cookie name" do
     it "sets a custom cookie name in the header" do
-      Clearance.configuration.cookie_domain = "custom_token"
```

Comment:
we are testing `cookie_name` configuration, but this was setting up `cookie_domain` configuration.